### PR TITLE
Removed unnecessary line in travis config

### PR DESCRIPTION
**Pull request by ohartl**
since php 5.3 was dropped in 70c3aa79fd56d630913c8534b35b44160303d4d6

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,7 +18,6 @@ before_script:
 script:
   - ./vendor/bin/phpunit --coverage-clover ./build/clover.xml
   - if [ $TRAVIS_PHP_VERSION != 'hhvm' ]; then php build/coverage-checker.php build/clover.xml 70; fi
-  - if [ $TRAVIS_PHP_VERSION != '5.3' ]; then ./vendor/bin/phpcs --standard=PSR2 ./src/ ./tests/; fi
```
Comment (Ocramius) [inline .travis.yml]: Should still run `./vendor/bin/phpcs`

Comment (gianarb) [inline .travis.yml]: No in this moment we don't run tests in PHP 5.3

Comment (Ocramius) [inline .travis.yml]: Yes, but PHPCS should still run anyway.

Comment (gianarb) [inline .travis.yml]: Right! :) I expect this fix and merge it

Comment (gianarb) [inline .travis.yml]: ping @ohartl :) Can you update this PR please? :)

Comment (ohartl) [inline .travis.yml]: @gianarb @Ocramius yep, my fault

**Comment by gianarb** [comment]
Thanks @ohartl

**Pull request merged**